src/appmixer/airtable/records/ListTables/transformers.js

```
@@ -0,0 +1,516 @@
+            case 'first':
+            case 'object':
+                if (addFields.includes('currentPageIndex')) {
+                    outputFields.unshift({ label: 'Current Record Index', value: 'index', schema: { type: 'integer' } });
+                }
+
+                if (addFields.includes('pagesCount')) {
+                    outputFields.unshift({ label: 'Records Count', value: 'count', schema: { type: 'integer' } });
+                }
+                return outputFields;
+            case 'array':
+                const arrayFields = transformSchemaToArray(outputFields);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)